apng-drawable/src/main/kotlin/com/linecorp/apng/RepeatAnimationCallback.kt

```diff
@@ -0,0 +1,17 @@
+package com.linecorp.apng
+
+/**
+ * This interface is for receiving callback when animation is about to be repeated.
+ */
+interface RepeatAnimationCallback {
+    /**
+     * This is called when animation is about to be repeated.
+     * [loopCount] is the total number of times the animation will be repeated.
+     * [nextLoop] is the loop count of the next animation.
+     */
+    fun onRepeat(
+        drawable: ApngDrawable,
+        loopCount: Int,
```

Comment:
We can get `loopCount` via `drawable.loopCount`, so I think it is redundant.

Reply:
DONE